<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vendor_window</class>
 <widget class="QMainWindow" name="vendor_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1229</width>
    <height>856</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>1111</width>
      <height>761</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>SEARCH PRODUCTS</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>921</width>
        <height>611</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>SEE ALL PRODUCT</string>
       </attribute>
       <widget class="QPushButton" name="refresh">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>30</y>
          <width>89</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>REFRESH</string>
        </property>
       </widget>
       <widget class="QScrollArea" name="scrollArea">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>80</y>
          <width>831</width>
          <height>461</height>
         </rect>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>829</width>
           <height>459</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>SEARCH A PRODUCT</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_100">
        <property name="geometry">
         <rect>
          <x>690</x>
          <y>30</y>
          <width>89</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>search</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>30</y>
          <width>421</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>130</y>
          <width>531</width>
          <height>431</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_2">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>130</y>
          <width>86</width>
          <height>431</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>product:-</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>upper limit:-</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>lower limit:-</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>ind limit:-</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>capacity:-</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>BUY PRODUCTS</string>
     </attribute>
     <widget class="QPushButton" name="refresh_vendor">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>70</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>REFRESH</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>160</y>
        <width>154</width>
        <height>381</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QLabel" name="label_26">
         <property name="text">
          <string>VENDOR NAME</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_27">
         <property name="text">
          <string>QTY:-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_28">
         <property name="text">
          <string>PRICE/KG</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_29">
         <property name="text">
          <string>PRICE:-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_30">
         <property name="text">
          <string>DATE OF BUY</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_31">
         <property name="text">
          <string>PRODUCT NAME</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>590</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>SUBMIT</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>150</y>
        <width>821</width>
        <height>411</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLineEdit" name="vendor_name"/>
       </item>
       <item>
        <widget class="QLineEdit" name="vendor_qty"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_2"/>
       </item>
       <item>
        <widget class="QLineEdit" name="vendor_price"/>
       </item>
       <item>
        <widget class="QDateEdit" name="dateEdit">
         <property name="minimumDate">
          <date>
           <year>2017</year>
           <month>3</month>
           <day>1</day>
          </date>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox"/>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>660</y>
        <width>841</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>check delivery status</string>
     </attribute>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>50</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>ENTER UNIQUE ID:-</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>40</y>
        <width>531</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="check_status">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>100</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>submit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>210</y>
        <width>701</width>
        <height>341</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu Condensed</family>
        <pointsize>23</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>close</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>190</y>
        <width>411</width>
        <height>191</height>
       </rect>
      </property>
      <property name="text">
       <string>close</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1229</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
